Composer/packages/lib/shared/src/settings.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { DialogSetting, PublishProfile } from '@botframework-composer/types';
+
+/**
+ * Apply a publishing profile to the base settings, resulting in a "ready to run" version of the settings
+ * for the new adaptive runtime. This function transforms the profile settings format to match the new
+ * Adaptive Runtime settings form.
+ * @param settings
+ * @param profile
+ * @returns
+ */
+export const applyPublishingProfileToSettings = (settings: DialogSetting, profile: PublishProfile): DialogSetting => {
+  // ensure the runtimeSettings key exists
+  settings.runtimeSettings = {
```

Comment:
Nit: You probably don't need to spread here.  settings.runtimeSettings = settings.runtimeSettings || {};